remediation-service/handler/problem_event_handler.go

```diff
@@ -1,7 +1,7 @@
 package handler
 
 import (
-	cloudevents "github.com/cloudevents/sdk-go"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
```

Comment:
I'm not sure if it's really that easy? Can I just upgrade cloudevents here or will this break our closed-loop-remediation?